homeassistant/components/sonos/__init__.py

```diff
@@ -225,6 +230,24 @@ def _async_add_visible_zones(subscription_succeeded: bool = False) -> None:
 
         async def async_subscription_failed(now: datetime.datetime) -> None:
             """Fallback logic if the subscription callback never arrives."""
+            addr, port = sub.event_listener.address
+            listener_address = f"{addr}:{port}"
+            if advertise_ip := soco_config.EVENT_ADVERTISE_IP:
+                listener_address += f" (advertising as {advertise_ip})"
+            ir.async_create_issue(
+                self.hass,
+                DOMAIN,
+                "subscriptions_failed",
```

Comment:
Shouldn't this be unique? As in, what if multiple sonos speakers raise the same error?